Files/UserControls/NavigationToolbar.xaml.cs

```diff
@@ -1241,6 +1241,7 @@ private void SearchButton_Click(object sender, RoutedEventArgs e)
             {
                 SearchRegion.Text = "";
                 IsSearchRegionVisible = false;
+                SearchButtonIcon.Glyph = "\uE721";
```

Comment:
Instead of changing the icon from code behind, can you use x:Bind?

Reply:
@yaichenbaum What's the advantage? Instead of changing the value, we are going to change the property, we have to manage an additional property.

Reply:
The advantage is that button isn't as tightly coupled to the code and therefor it will be easier to work with both the xaml and code.

Reply:
Ok. I do it tomorrow.

Reply:
It is done.